packages/dds/tree/.eslintrc.cjs

```diff
@@ -70,6 +70,12 @@ module.exports = {
 				"@typescript-eslint/explicit-function-return-type": "off",
 			},
 		},
+		{
```

Comment:
seems worth augmenting the note on where this is enabled or duplicating the comment here

```ts
// TODO: Remove this override once this config has been updated to extend at least the "recommended" base config.
```

Reply:
```suggestion
		// TODO: Remove this override once this config has been updated to extend at least the "recommended" base config.
		{
```